### PR TITLE
BZ1913826

### PR DESCRIPTION
**Pull request by kquinn1204**
This is the original bug https://bugzilla.redhat.com/show_bug.cgi?id=1913826

### CODE REVIEW

**Comment by kquinn1204** [comment]
@aireilly please review

**Comment by kquinn1204** [comment]
@abhatt-rh, please review

**Comment by kquinn1204** [comment]
@ahardin-rh Adding known issue in https://bugzilla.redhat.com/show_bug.cgi?id=1913826. The link called out in the text is dependent on https://github.com/openshift/openshift-docs/pull/28203

**Comment by kquinn1204** [comment]
@ahardin-rh The description of this issue matches closely that in Bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1913826 with the exception of the addition of the error. 
link is https://deploy-preview-28739--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues

The description starts: Upgrading Performance Addon Operator
Guess it is ready for merging.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -612,6 +612,16 @@ This script removes unauthenticated subjects from the following cluster role bin
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
 * {op-system} real time (RT) kernels are currently only supported on compute nodes, not control plane nodes. Compact clusters are not supported with RT kernels in {product-title} 4.7. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887007[*BZ#1887007*])
+* Upgrading Performance Addon Operator from 4.6 to 4.7 fails with the error:
++
+[source,terminall]
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:617]: terminal

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -612,6 +612,16 @@ This script removes unauthenticated subjects from the following cluster role bin
...
+----
+"Warning  TooManyOperatorGroups  11m   operator-lifecycle-manager  csv created in namespace with multiple operatorgroups, can't pick one automatically"
+----
++
+Before upgrading follow the procedure as described in xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#upgrading-performance-addon-operator-configured-for-a-specific-namespace_cnf-master[Upgrading Performance Addon Operator when previously installed to a specific namespace].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:622]: upgrading, follow

**Comment by ahardin-rh** [comment]
@kquinn1204 Looks great! Just some minor style suggestions from me and we should be set. Thanks for this!

**Pull request merged**